rllib/core/optim/tests/test_rl_optimizer.py

```diff
@@ -0,0 +1,378 @@
+import gym
```

Comment:
Fine to leave these classes here for this PR. But we should soon move these into the `examples` folder. When we start documenting these new APIs.